library/auth.inc

```diff
@@ -84,7 +84,11 @@ if (
     // Logout
     // If session has timed out / been destroyed, logout record for null user/provider will be invalid.
```

Comment:
hi @bradymiller, so the log will now indicate that it was a successful logout (timeout)?

Reply:
yes, for now, but this may not be ideal. let me know if think of a better way to log this (ie. should i make it a 'timeout' instead of a 'logout' for the action)?